### PR TITLE
CI: Test installations

### PR DESCRIPTION
**Pull request by W95Psp**
This PR:
 - adds installation tests for (latest and old) Ubuntu and Mac (`opam` on Ubuntu 20.04 is quite old)
 - fixes `setup.sh`
 - configure bors

### CODE REVIEW

**Comment by W95Psp** [comment]
bors try

**Comment by W95Psp** [comment]
bors try

**Comment by W95Psp** [comment]
bors try-

**Comment by W95Psp** [comment]
bors try

**Comment by W95Psp** [comment]
bors try-

**Comment by W95Psp** [comment]
bors try-

**Comment by W95Psp** [comment]
bors try

File: bors.toml
Comment (franziskuskiefer) [inline bors.toml:1]: Can you add some doc somewhere to a readme such that everyone know how to handle bors?

File: CI.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   FStar, we do not run Coq or FStar on the extractions.
+ - [`test_installs`](./.github/workflows/test_installs.yml): compiles
+   the toolchain on two versions of Ubuntu and two versions of MacOS
+   using `apt` or `homebrew` and the `setup.sh` script.
+ 
+## Pull Requests & Bors
+Each commit pushed to a pull request triggers the actions `specs` and
+`format`.
+
+When a PR is reviewed and ready to be merged, one should not press the
```
Comment (franziskuskiefer) [inline CI.md:26]: Can you make this a required job such that it needs to run before merging?

**Comment by W95Psp** [comment]
bors +r

**Comment by W95Psp** [comment]
bors try

**Comment by W95Psp** [comment]
bors +r

**Pull request merged**